Compile Error: non-deducible template parameter 'Args'

### Error Message
> make[4]: *** [CMakeFiles/tox4j-c.dir/src/tox/core.cpp.o] Error 1
[ 48%] Building CXX object CMakeFiles/tox4j-c.dir/src/util/debug_log.cpp.o
In file included from /Users/ttd/jvm-toxcore-c/cpp/src/util/debug_log.cpp:3:
In file included from /Users/ttd/jvm-toxcore-c/cpp/src/tox/core.h:3:
/Users/ttd/jvm-toxcore-c/cpp/src/tox/common.h:51:12: error: class
      template partial specialization contains a template parameter that cannot be
      deduced; this partial specialization will never be used
      [-Wunusable-partial-specialization]
    struct mk_tox_cb<Subsystem, void (Subsystem *, Args..., void *)>
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/ttd/jvm-toxcore-c/cpp/src/tox/common.h:50:46: note: 
      non-deducible template parameter 'Args'
    template<typename Subsystem, typename ...Args>

### Compiler

> Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -175,6 +175,11 @@ target_link_libraries(${PROJECT_NAME}
   ${PROTOBUF_LIBRARIES}
 )
 
+target_compile_options(${PROJECT_NAME}
+  PRIVATE
+    -Wno-error=unusable-partial-specialization
+)
+
 if(LIBTOXCORE_FOUND)
   target_link_libraries(${PROJECT_NAME}
     ${LIBTOXAV_STATIC_LIBRARIES}

--- a/project/build.properties
+++ b/project/build.properties
@@ -1,1 +1,1 @@
-sbt.version=0.13.12
+sbt.version=0.13.18

